mail_tracking/views/mail_tracking_email_view.xml

```diff
@@ -22,7 +22,7 @@
                     </group>
                     <group>
                         <group>
-                            <field name="mail_message_id" />
+                            <field name="mail_message_id" string="Message" />
```

Comment:
String should be at model level preferably

Reply:
This was chosen because in `v15`, these two fields have the same label:

https://github.com/OCA/social/blob/09f73b72c9a9f6829213bd500fb2ff7dd0d2c292/mail_tracking/models/mail_tracking_email.py#L46-L49

which triggers a warning at installation. Since this is the only view in the whole module using this field, I chose to add the string here directly to preserve the original `v15` look. Would you rather have the string removed and show "Mail Message" instead of "Message"? I admit I am not a fan of modifying labels directly on views either!